src/state/trajectory/logics.ts

```diff
@@ -345,6 +353,104 @@ const loadFileViaUrl = createLogic({
     type: LOAD_FILE_VIA_URL,
 });
 
+// configures the controller for file conversion and sends server health checks
+const configureFileConversionLogic = createLogic({
+    process(
+        deps: ReduxLogicDeps,
+        dispatch: <T extends AnyAction>(action: T) => T,
+        done
+    ) {
+        const { getState } = deps;
+
+        // TODO: Most likely this will eventually replace the config up top
+        // but for development purposes it's here
+        // until we switch to Octopus, make sure it matches your local instance.
+        const netConnectionConfig: NetConnectionParams = {
+            serverIp: "0.0.0.0",
+            serverPort: 8765,
+            useOctopus: true,
+            secureConnection: false,
```

Comment:
Do these need to be set back afterwards to enable someone to load a networked trajectory?

Reply:
I think in production we won't need to reset these because everything will use the same Octopus backend, and we will be able to remove this config and just use the same one for everything stored at the top of the file.

Reply:
have you tested initiating the config and then trying to load a networked file afterwards?

Reply:
i did although i realize now it needs a little more work.
what i tested that works: currently if you navigate to `/import` and then to a trajectory url like `/viewer?trajFileName=endocytosis.simularium` it works.

but i now realize that, navigating to that url with the back button errors out, and if you hit `/import` then quickly navigate away while the server check is till pending there are some status setting side effects (i.e. the load trajectory button gets disabled while on the homepage for example) so i think i need to make this a bit more sophisticated in the logic and the router.